internal/validations/validations.go

```diff
@@ -8,6 +8,25 @@ import (
 	"github.com/distribution/distribution/reference"
 )
 
+var (
+	defaultAllowAll = map[string]struct{}{"*": {}}
+	imageURIs       = defaultAllowAll
+)
+
+// SetImageURIs restricts the approved container URIs to the provided set. To reset to a default allow-all state,
+// provide an empty list.
+func SetImageURIs(uris []string) {
```

Comment:
Probably would actually plant this behind an Init func or NewValidator depending on what we chose to take on. Would ponder the whole empty == default thing too... that was def a quick add to make tests easy.